core/diagnostics/DiagnosticScenarios/readme.md

```diff
@@ -4,13 +4,13 @@ languages:
 products:
 - dotnet-core
 page_type: sample
-name: "Diagnostic Scenarios"
+name: "DiagnosticScenarios sample debug target"
 urlFragment: "diagnostic-scenarios"
 description: "A .NET Core sample with methods that trigger undesirable behaviors to diagnose."
 ---
 # DiagnosticScenarios sample debug target
```

Comment:
This heading isn't rendered as shown in the screenshot. so, I updated the "name" in the metadata section.

Reply:
@mairaw, @BillWagner, Should the h1 be completely deleted as they're not rendered at all ?

Reply:
I'd have to investigate. I haven't looked at the samples guidance yet.